lib/compat/wordpress-6.9/rest-api.php

```diff
@@ -29,3 +29,29 @@ function gutenberg_rest_theme_export_link_rel( $response, $theme ) {
 	return $response;
 }
 add_filter( 'rest_prepare_theme', 'gutenberg_rest_theme_export_link_rel', 10, 2 );
+
+
+/**
+ * Adds edit link relation to the attachment responses.
+ *
+ * @param WP_REST_Response $response   The response object.
+ * @param WP_Post          $attachment Attachment object used to create response.
+ * @return WP_REST_Response Modified   response object.
+ */
+function gutenberg_rest_attachment_edit_link_rel( $response, $attachment ) {
+	if ( $attachment->post_type === 'attachment' ) {
+		$links = $response->get_links();
+		$edit_link_rel = 'https://api.w.org/duplicate-modified';
```

Comment:
I'll roll this back. Now that I think of it, `/media/edit` is pretty quirky, and maybe shouldn't announce itself to the world just yet until we standardize some type of behaviour.